modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/service/test/DLFileEntryLocalServiceTest.java

```diff
@@ -1253,6 +1260,60 @@ public void testUpdateDisplayDateExpirationDateReviewDate()
 		Assert.assertEquals(reviewDate, dlFileEntry.getReviewDate());
 	}
 
+	@Test
+	public void testUpdateFileEntryDynamicQueryTransactionInterval()
+		throws Exception {
+
+		List<DLFileEntry> dlFileEntries = new ArrayList<>();
```

Comment:
Why are you adding the file entries to a list? You're not using it later.

Reply:
These is a remain of the old count I made during my tests, removed.